src/components/atoms/Timeline/index.tsx

```diff
@@ -22,10 +24,25 @@ export type Props = {
   range?: { [K in keyof Range]?: Range[K] };
   onClick?: TimeEventHandler;
   onDrag?: TimeEventHandler;
+  onPlay?: TimeEventHandler;
+  onOpen?: () => void;
+  onClose?: () => void;
+  isOpened?: boolean;
+  publishedTheme?: PublishTheme;
```

Comment:
I am not sure really the plan for HOW we are going to use this component. If it'll be different from the builtin Widgets then perhaps this is fine. But if it'll be treated like a builtin widget this will have to be replaced with sceneProperty and then the PublishedTheme will be retrieved from here, just like the Storybook, Button, etc.
For now lets leave this as is since maybe that hasn't been decided yet.

Reply:
Timeline component will be used as builtin widget. So I fixed.
[fix: use sceneProperty](https://github.com/reearth/reearth-web/pull/277/commits/c4dd273590f4df91bad30bc33c2a89b101be9369)